### DISCUSSION TITLE
Disable rodauth create account route for Admin

### DISCUSSION THREAD

**Discussion by jaahoo** [janko/rodauth-rails#110] [Q&A] [answered]
Hi,
I use rodauth-rails for User and Admin login. I set multiple configurations.
in _rodauth_app.rb_ I have
`r.rodauth`
`r.rodauth(:admin)`
And in _rodauth_admin.rb_ I enable account creation `enable :create_account`
BUT I don't want to be able to create an admin account via the form. I want to create an admin only from rails console.
If I don't enable create_account I can't create account via console with
`RodauthApp.rodauth(:admin).create_account(login: "user@example.com", password: "secret")`
So is there an option to disable create_account_path route?
Or is my approach completely wrong and there is a better solution?
Thank you for any help and keep great work!

**Comment by janko** [comment, marked as answer]
I believe the way to disable routes is to set the corresponding `*_route` value to `nil`:

```rb
create_account_route nil
```

If this works and the internal request remains functional, perhaps it would be worth documenting this in an official guide (perhaps "[Change route path](http://rodauth.jeremyevans.net/rdoc/files/doc/guides/paths_rdoc.html)").

**Reply by jaahoo** [reply]
Thank you. This works!

**Reply by jaahoo** [reply]
I created a wiki page
https://github.com/janko/rodauth-rails/wiki/Disable-predefined-Rodauth-routes

**Reply by janko** [reply]
I ended up documenting it in the official Rodauth guide – https://github.com/jeremyevans/rodauth/pull/245. So, I'll remove the wiki page, because this is not functionality that's provided by rodauth-rails.
